drivers/SmartThings/zwave-switch/src/test/test_qubino_flush_2_relay.lua

```diff
@@ -17,124 +17,487 @@ local t_utils = require "integration_test.utils"
 local capabilities = require "st.capabilities"
 local zw = require "st.zwave"
 local zw_test_utils = require "integration_test.zwave_test_utils"
-local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({version=2})
-local Meter = (require "st.zwave.CommandClass.Meter")({version=3})
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 1 })
+local Meter = (require "st.zwave.CommandClass.Meter")({ version = 3 })
+local SensorMultilevel = (require "st.zwave.CommandClass.SensorMultilevel")({ version = 7 })
 
--- supported comand classes: SWITCH_BINARY
 local qubino_flush_2_relay_endpoints = {
   {
     command_classes = {
-      {value = zw.SWITCH_BINARY},
-      {value = zw.METER},
+      { value = zw.SWITCH_BINARY },
+      { value = zw.METER },
+      { value = zw.SENSOR_MULTILEVEL }
+    }
+  },
+  {
+    command_classes = {
+      { value = zw.SWITCH_BINARY },
+      { value = zw.METER },
+    }
+  },
+  {
+    command_classes = {
+      { value = zw.SWITCH_BINARY },
+      { value = zw.METER },
+    }
+  },
+  {
+    command_classes = {
+      { value = zw.SENSOR_MULTILEVEL }
     }
   }
 }
 
-local mock_qubino_flush_2_relay = test.mock_device.build_test_zwave_device({
-  profile = t_utils.get_profile_definition("qubino-flush2-relay-temperature.yml"),
-  zwave_endpoints = qubino_flush_2_relay_endpoints
+local parent_profile = t_utils.get_profile_definition("qubino-flush2-relay.yml")
+local child_relay_profile = t_utils.get_profile_definition("metering-switch.yml")
+local child_temperature_profile = t_utils.get_profile_definition("child-temperature.yml")
+
+local mock_parent_device = test.mock_device.build_test_zwave_device({
+  profile = parent_profile,
+  zwave_endpoints = qubino_flush_2_relay_endpoints,
+  zwave_manufacturer_id = 0x0159,
+  zwave_product_type = 0x0002,
+  zwave_product_id = 0x0051
+})
+
+local mock_child_2_device = test.mock_device.build_test_child_device({
+  profile = child_relay_profile,
+  device_network_id = string.format("%s:%02X", mock_parent_device.device_network_id, 2),
```

Comment:
This is not what the `DNI` will be within edge drivers.  the `device_network_id` will just be a UUID and you shouldn't include it in these templates.

Reply:
done